hack/functest.sh

```diff
@@ -40,7 +40,7 @@ find . -name .kubeconfig || true
 KUBE_CONFIG=$(${KUBEVIRTCI_PATH}/kubeconfig.sh)
 
 
-TEST_NAMESPACE=node-maintenance-operator go test ./test/e2e/... -root=$(pwd) -kubeconfig=${KUBE_CONFIG} -globalMan _out/nodemaintenance_crd.yaml --namespacedMan _out/namespace-init.yaml -singleNamespace
+GO111MODULE=on TEST_NAMESPACE=node-maintenance-operator go test ./test/e2e/... -root=$(pwd) -kubeconfig=${KUBE_CONFIG} -globalMan _out/nodemaintenance_crd.yaml --namespacedMan _out/namespace-init.yaml -singleNamespace
```

Comment:
please use `GOFLAGS=-mod=vendor` here as well

Reply:
that didn't go well: build gopkg.in/yaml.v2: cannot find module for path gopkg.in/yaml.v2

Reply:
go test -mod=vendor works, however GOFLAGS env variable doesn't.

Reply:
strange things happen at sea.

with 
```
GO111MODULE=on TEST_NAMESPACE=node-maintenance-operator go test -v -mod=vendor ./test/e2e/... -root=$(pwd) -kubeconfig=${KUBE_CONFIG} -globalMan _out/nodemaintenance_crd.yaml --namespacedMan _out/namespace-init.yaml -singleNamespace
```
I get 
```
build gopkg.in/yaml.v2: cannot find module for path gopkg.in/yaml.v2
Makefile:112: recipe for target 'cluster-functest' failed
```
even though we have 
```
+ pwd
/home/prow/go/src/github.com/kubevirt/go/src/kubevirt.io/node-maintenance-operator
+ find . vendor -type d
./kubevirtci/pack8s/vendor/gopkg.in/yaml.v2
```

With 
```
TEST_NAMESPACE=node-maintenance-operator
+ go test -mod=vendor ./test/e2e/... -root=/home/prow/go/src/github.com/kubevirt/go/src/kubevirt.io/node-maintenance-operator -kubeconfig=/home/prow/go/src/github.com/kubevirt/node-maintenance-operator/kubevirtci/_ci-configs/k8s-1.15/.kubeconfig -globalMan _out/nodemaintenance_crd.yaml --namespacedMan _out/namespace-init.yaml -singleNamespace 
```
we get 
```
build flag -mod=vendor only valid when using modules
```


With 
```
TEST_NAMESPACE=node-maintenance-operator go test -v ./test/e2e/... -root=$(pwd) -kubeconfig=${KUBE_CONFIG} -globalMan _out/nodemaintenance_crd.yaml --namespacedMan _out/namespace-init.yaml -singleNamespace
```
the tests pass. Note that I added the -v option, and the logs don't show that it builds the stuff all over